shared-bindings/_pixelbuf/PixelBuf.c

```diff
@@ -349,23 +332,14 @@ STATIC mp_obj_t pixelbuf_pixelbuf_unary_op(mp_unary_op_t op, mp_obj_t self_in) {
 
 //|   .. method:: show()
 //|
-//|     Call the associated write function to display the pixels.
+//|     Does nothing unless subclassed.
```

Comment:
Think we should raise a `NotImplementedError`?

Reply:
Should we add another error string, or is `mp_raise_NotImplementedError(NULL)` ok here?

Reply:
NULL is fine.